### PR TITLE
feat: access onshore and offshore wind profiles

### PR DESCRIPTION
**Pull request by kasparm**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Retrieve onshore and offshore wind profile for a scenario 

### What the code is doing
Two functions are implemented:
* plant ID are retrieved for each type of turbines 
* wind profile is queried
* an error is raised if no offshore wind profiles id found in the grid

### Testing
Manual testing for scenario.
Please propose potential test.

### Where to look
Check the `powersimdata.scenario.analyze` module

### Usage Example/Visuals
```
from powersimdata import Scenario

scenario = Scenario(600)

offshore_wind = scenario.get_wind_offshore()
onshore_wind = scenario.get_wind_onshore()
```

### Time estimate
10 min

### CODE REVIEW

File: powersimdata/scenario/analyze.py [hunk 1]
```diff
@@ -323,6 +323,36 @@ def get_wind(self):
...
+ 
+        grid = self.get_grid()
+        onshore_id = grid.plant.groupby(["type"]).get_group("wind").index
+        return wind[onshore_id]
+
+    def get_wind_offshore(self):
+        """Returns wind offshore profile
+ 
+        :return: (*pandas.DataFrame*) -- data frame of wind energy output for offshore
+            turbines
```
Comment (rouille) [inline powersimdata/scenario/analyze.py:344]: Missing entry in docstring for the raised error. You should add: `:raises ValueError: if no offshore wind turbinee is in the grid`

Comment (kasparm) [reply powersimdata/scenario/analyze.py:344]: done

**Comment by rouille** [comment]
I have tested that the error is raised when there is no offshore wind turbine:
```
>>> from powersimdata import Scenario
>>> 
>>> scenario = Scenario(403)
Failed to download ScenarioList.csv from server
Falling back to local cache...
Failed to download ExecuteList.csv from server
Falling back to local cache...
SCENARIO: test | EasternBase_2020_3

--> State
analyze
--> Loading grid
Loading bus
Loading plant
Loading heat_rate_curve
Loading gencost_before
Loading gencost_after
Loading branch
Loading dcline
Loading sub
Loading bus2sub
Failed to download ScenarioList.csv from server
Falling back to local cache...
--> Loading ct
>>> offshore_wind = scenario.get_wind_offshore()
--> Loading wind
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/brdo/CEM/PowerSimData/powersimdata/scenario/analyze.py", line 356, in get_wind_offshore
    raise ValueError("No offshore wind turbines in grid")
ValueError: No offshore wind turbines in grid
>>> onshore_wind = scenario.get_wind_onshore()
--> Loading wind
>>> wind = scenario.get_wind()
--> Loading wind
>>> wind.shape
(8784, 640)
>>> onshore_wind.shape
(8784, 640)
```

**Comment by BainanXia** [comment]
Moving related conversation from Slack here:

@BainanXia  I’m looking at analyze.py and wondering whether we could re-use OutputData call and TransformProfile call in a singe object instantiation.

@rouille InputData and OutputData could definitely be attributes of the Analyze but I would say it is out of the scope of this PR

**Review by rouille — APPROVED** [review]
Nice

**Pull request merged**